gallery/static/js/gallery.js

```diff
@@ -315,6 +315,36 @@ function moveDir() {
     });
 }
 
+function toggleFilePin(pinned, this_id) {
+    if (pinned) {
+        $.ajax({
+            type: "POST",
+            url: "/api/file/pin/" + this_id,
+        });
+    } else {
+        $.ajax({
+            type: "POST",
+            url: "/api/file/unpin/" + this_id,
+        });
+    }
+    location.reload();
+}
+
+function toggleDirPin(pinned, this_id) {
```

Comment:
Honestly, why not just make a pin and unpin function rather than making it an arg, since it's not like any call uses a variable for the arg